updater/lib/dependabot/environment.rb

```diff
@@ -31,6 +31,10 @@ def self.github_actions?
       @github_actions ||= environment_variable("GITHUB_ACTIONS", false)
     end
 
+    def self.job_definition
+      @job_definition ||= JSON.parse(File.read(job_path))
+    end
```

Comment:
💡 Not sure if this would be supported by our current code, but I think it'd be a big benefit to deep freeze this object.

Reply:
Yeah, that's a fair call out. I'd like to find a more durable home for the read from/write to file methods in general, so I'll keep this in mind.